src/structlog/tracebacks.py

```diff
@@ -83,10 +83,14 @@ class SyntaxError_:  # noqa: N801
 class Stack:
     """
     Represents an exception and a list of stack frames.
+
+    .. versionchanged:: 24.5.0
+       Added the *exc_notes* field.
     """
 
     exc_type: str
     exc_value: str
+    exc_notes: str | None = None
```

Comment:
hey sorry for the long delay – is there a reason why you chose to make it an optional str?

You mention it in the description but without reasoning and I think it would be more ergonomic to make it a `tuple[str, ...]`?

(tuple > list because it would keep Stack hashable)

Reply:
I forget; I think I was concerned about how it would show up in a particular system we have, but for no good reason. A tuple works better. Done! Sorry I lost track of this for a while.

Reply:
thanks! but ugh I'm so dumb, there already _is_ a list in the class so it makes no sense to make this inconsistently a list. I'll fix it myself, this is on me.